python/pyspark/mllib/fpm.py

```diff
@@ -41,8 +41,11 @@ class FPGrowthModel(JavaModelWrapper):
     >>> model = FPGrowth.train(rdd, 0.6, 2)
     >>> sorted(model.freqItemsets().collect())
     [FreqItemset(items=[u'a'], freq=4), FreqItemset(items=[u'c'], freq=3), ...
+
+    .. addedversion:: 1.4.0
```

Comment:
To be consistent with the `pyspark.sql` module we ought to use 2-part version numbers (i.e. `1.4`).